doc/source/ray-core/doc_code/direct_transport_gloo.py

```diff
@@ -106,6 +106,7 @@ def sum(self, tensor_dict: dict):
 # __gloo_intra_actor_start__
 import torch
 import ray
+import pytest
```

Comment:
Why doe we need this import now? The only change in this file is to a print statement....